src/app.py

```diff
@@ -12,10 +12,19 @@
 app = FastAPI()
 
 
-@app.get("/")
-def healthCheck():
-    return {"status": True, "internalCode": InternalCode.IC_GENERIC_SUCCESS}
+@app.get(
+    "/", status_code=status.HTTP_200_OK, response_description="Welcome to MySK API"
+)
+def healthCheck(response: Response):
```

Comment:
Is the function name here required to look exactly like this? If not, this should probably be snake_case.